dashboard/dashboard/styles.py

```diff
@@ -68,9 +68,9 @@
     "--text-field-selection-color": "",
     "--text-field-focus-color": "transparent",
     "--text-field-border-width": "1px",
-    "background-clip": "content-box",
-    "background-color": "transparent",
-    "box-shadow": "inset 0 0 0 var(--text-field-border-width) transparent",
+    "backgroundClip": "content-box",
+    "backgroundColor": "transparent",
+    "boxShadow": "inset 0 0 0 var(--text-field-border-width) transparent",
```

Comment:
```suggestion
    "background_clip": "content-box",
    "background_color": "transparent",
    "box_shadow": "inset 0 0 0 var(--text-field-border-width) transparent",
```

we prefer python-style snake case for these css keys (they get converted to camelCase)